application/backend/src/main/resources/apps/typerefinery/components/layout/containerlist/variant.html

```diff
@@ -0,0 +1,26 @@
+<template data-sly-template.variant="${ @ model }">
+    <div 
+      component="${model.componentName}"
+      id="${model.id}"
+      class="${model.componentClassNames}" 
+      data-model="${model.jsonString}"
+      data-sly-element="${model.decorationTagName}"
+      >
+    </div>
+    <sly data-sly-call="${containerListTemplate @ model=model}"></sly>
+</template>
+<template data-sly-template.containerListTemplate="${ @ model }">
+  <script id="${model.id}-template" type="text/x-handlebars-template">
+    <div id="containerItems" class="containerItems {{alignment}}">
```

Comment:
you can have this attr to UL instead of div

Reply:
Done